packages/notebook/test/tracker.spec.ts

```diff
@@ -1,22 +1,31 @@
 // Copyright (c) Jupyter Development Team.
-// Distributed under the terms of the Modified BSD License.
```

Comment:
This probably was an accidental deletion.

Reply:
I've added this to https://github.com/jupyterlab/jupyterlab/issues/8327 as a follow-on task for all test files.